source/reference/auth-plugin-jdbc.txt

```diff
@@ -14,18 +14,20 @@ JDBC Authentication Plugin
 
 .. _install-auth-jdbc-plugin:
 
-.. important::
+Prerequisites
+-------------
 
-   The plugin is not compatible with MySQL Server or Connector/J
-   driver version 8 and later.
+This plugin:
+
+- Requires :mvn:`MySQL Connector/J 5.1.39 </mysql/mysql-connector-java/5.1.39>` or later.
```

Comment:
Why does this link point to maven, while the link in the Maven section points to a tar / zip download?